src/views/CardStack.js

```diff
@@ -189,18 +182,27 @@ class CardStack extends Component<DefaultProps, Props, void> {
     transitionProps: NavigationTransitionProps,
     headerMode: HeaderMode
   ): ?React.Element<*> {
-    const headerConfig = this.props.router.getScreenConfig(
+    const headerConfig: HeaderConfig = this.props.router.getScreenConfig(
       transitionProps.navigation,
       'header'
     ) || {};
 
+    if (
+      headerConfig.bar === null
```

Comment:
I think this logic should be `typeof headerConfig.bar !== 'undefined' && typeof headerConfig.bar !== 'function'`